src/coreclr/jit/instrsxarch.h

```diff
@@ -611,8 +611,8 @@ INST3(FIRST_AVX512_INSTRUCTION, "FIRST_AVX512_INSTRUCTION", IUM_WR, BAD_CODE, BA
 INST3(kmovw_gpr,        "kmovw",            IUM_WR, BAD_CODE,               BAD_CODE,     PCKFLT(0x92),                  INS_TT_NONE,                                          REX_W0                       | Encoding_VEX                                                                                                   | KInstruction)
 INST3(kmovw_msk,        "kmovw",            IUM_WR, PCKFLT(0x91),           BAD_CODE,     PCKFLT(0x90),                  INS_TT_NONE,                                          REX_W0                       | Encoding_VEX                                                                                                   | KInstruction)
 INST3(kortestw,         "kortestw",         IUM_WR, BAD_CODE,               BAD_CODE,     PCKFLT(0x98),                  INS_TT_NONE,                                          REX_W0                       | Encoding_VEX | Resets_OF      | Resets_SF     | Writes_ZF     | Resets_AF     | Resets_PF     | Writes_CF      | KInstruction)
-INST3(vextractf64x4,    "extractf64x4",     IUM_WR, SSE3A(0x1B),            BAD_CODE,     BAD_CODE,                      INS_TT_TUPLE4,                       Input_64Bit    | REX_W1_EVEX                  | Encoding_EVEX  | INS_Flags_IsDstDstSrcAVXInstruction)                                                                                           // Extract 256-bit packed double-precision floating point values
```

Comment:
`DstDstSrc`, `DstSrcSrc`, and others are somewhat "poor" names.

They originally existed because we didn't have proper VEX support and everything was considered RMW. So, we went from encoding things in `ModRM:reg` and `ModRM:r/m` into also having a new `VEX.vvvv` "field".

Since the vast majority of instructions were of the form `ins reg, r/m` this was basically used to indicate whether we duplicated the `Dst` (`ModRM:reg`) or `Src` (`ModRM:r/m`) register into the `VEX.vvvv` field. -- This "worked" because we also didn't really support containment for stores at all, so we never really had to care about `ModRM:r/m` being the `Dst`.

Today, we do support VEX properly, don't require everything to be viewed as `RMW`, and do also support generating stores for HWIntrinsics. So, it might be good to rethink the flag names a bit in the future. Namely, we really just care about which operands go into which of the three `ModRM:reg`, `VEX.vvvv`, and `ModRM:r/m` (with that particular order for `Dst, Op1, Op2` being the most common).